MaxText/layers/attentions.py

```diff
@@ -365,7 +365,7 @@ def generate_attention_mask(
       all_ones = jnp.ones_like(output_mask)
       sliding_mask = jnp.triu(all_ones, -1 * self.sliding_window_size + 1) * jnp.tril(all_ones, self.sliding_window_size - 1)
       output_mask = sliding_mask * output_mask
-    elif self.attention_type == AttentionType.CHUNK and output_mask is not None:
+    elif self.attention_type == AttentionType.CHUNK and output_mask is not None and output_mask.shape[-1] > self.chunk_attn_window_size:
```

Comment:
Tiny nit but this line is a bit rough. Could be mitigated by hoisting the last expression to a variable and could import AttentionType